pub_sub/javascript/http/order-processor/index.js

```diff
@@ -1,11 +1,8 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 
-const APP_PORT = process.env.APP_PORT
-if(!APP_PORT) {
-    console.log('[error]: --app-port is not set. Re-run dapr run with -p or --app-port.\nUsage: https://docs.dapr.io/getting-started/quickstarts/pubsub-quickstart/\n');
-    process.exit(1);
-}
+const APP_PORT = process.env.APP_PORT || 5001;
```

Comment:
```suggestion
const APP_PORT = process.env.APP_PORT ?? '5001';
```

Prefer we use ?? operator which covers more cases for false that come up in Javascript.  I. believe we want port to be a string too.